src/contracts/auth.config.ts

```diff
@@ -0,0 +1,8 @@
+export interface IAuthConfig {
```

Comment:
Do not prefix interface names.

Reply:
Resolved in 27df0236a92a4ee09355ebf06d8819a8fefd3c42.